main.bal

```diff
@@ -1169,6 +1169,40 @@ service / on new http:Listener(serverPort) {
         }
         check caller->respond(resp);
     }
+
+    # Update template id of an experiment.
+    #
+    # + return - an empty response with a 2xx http status code on success
```

Comment:
I normally prefer returning the updated value for post and put requests (same output as would be returned by a get). This allows the client to not guess the new value on success (or having to query the get endpoint immediately to be sure).